lib/pbench/test/unit/agent/task/test_results_push.py

```diff
@@ -79,12 +107,9 @@ def test_bad_arg():
             ],
         )
         assert result.exit_code == 2
-        assert (
-            result.stderr.find(
-                "Invalid value for 'RESULT_TB_NAME': "
-                "File 'nothing.tar.xz' does not exist."
-            )
-            > -1
+        assert result.stderr.find(
+            "Invalid value for 'RESULT_TB_NAME': "
+            "File 'nothing.tar.xz' does not exist."
```

Comment:
I don't think that removing the `> -1` will do what you want:  this assertion will now never fail, unless the searched-for string occurs at the beginning of the stream.  (That is, it will _pass_ if the string is not found, or if the string is found anywhere else.)

I think you want something more like
```python
        assert (
            "Invalid value for 'RESULT_TB_NAME': "
            "File 'nothing.tar.xz' does not exist." in result.stderr
        )
```

Reply:
I didn't even realize that I did that.